charts/perses/Chart.yaml

```diff
@@ -3,8 +3,8 @@ name: perses
 description: Perses helm chart
 icon: https://avatars.githubusercontent.com/u/77209215?s=200&v=4
 type: application
-version: 0.8.0
-appVersion: "v0.49.0"
+version: 0.9.0
+appVersion: "v0.50.0"
```

Comment:
Maybe worth to split in different prs?

Reply:
Sure! Just to clarify. You would like to bump the appVersion in a new PR, right? 😃